src/Client/Client.php

```diff
@@ -55,13 +56,24 @@ public function createRequest(Request $request)
      *
      * @param \Assertis\Http\Request\Request $request
      * @return ResponseInterface
+     * @throws Exception
      */
     public function send(Request $request)
     {
         try {
             $response = $this->guzzleClient->send($this->createRequest($request));
-        } catch (RequestException $exception) {
-            $response = $exception->getResponse();
+        } catch (Exception $exception) {
+            if ($exception instanceof RequestException) {
```

Comment:
Why don't just: 
catch(RequestException $e){
if (empty($e->getResponse())
}

i don't get it....